guide.md

```diff
@@ -0,0 +1,136 @@
+# HIRO 7 JS Migration Guide (working draft)
+
+## Libraries
+
+### HIRO Graph Client
+
+`@hiro-graph/client@^7.3.9` (https://www.npmjs.com/package/@hiro-graph/client)
+
+```JS
+import Client from '@hiro-graph/client';
+
+const client = new Client({
+  endpoint: 'http://localhost:8888',
+  token: tokenInstance,
+});
+```
+
+## Creating, Deleting Nodes and Relationships
+
+### Node Creation
+
+For the node creation use the create method from the hiro graph client.
+
+Example:
+
+```JS
+const dataForHiro = {
+  ...,
+  'ogit/_scope': instanceScopeId
+};
+
+return client.create(OGIT_NODE_TYPE, dataForHiro);
+```
+
+Where:
+
+- client: is the client from the library.
+- OGIT_NODE_TYPE: is the node type being created.
+- dataForHiro: is a JSON object containing the data needed for the creation of the node.
+
+### Node Deletion
+
+For the node deletion use the delete method from the hiro graph client.
+
+Example:
+
+```JS
+return client.delete(nodeId);
+```
+
+Where:
+
+- client: is the client from the library.
+- nodeId: is the ID of the node being deleted.
+
+### Relations Creation
+
+For creating and removing relationship between nodes use the connect and disconnect methods from hiro graph client.
+
+```JS
+client.connect(OGIT_EDGE_TYPE, inId, outId);
+```
+
+```JS
+client.disconnect(OGIT_EDGE_TYPE, inId, outId);
+```
+
+Where:
+
+- client: is the client from the library.
+- OGIT_EDGE_TYPE: is edge relationship type being created or disconnected.
+- inId: is the Node id of the IN relationship
+- outId: is the Node id of the OUT relationship
+
+## Queries
+
+### Gremlin
+
+_Gremlin is now part of the `@hiro-graph/client` library_
+
+```JS
+import { gremlin } from '@hiro-graph/client';
+
+const query = gremlin('')
+  .inE(OGIT_EDGE_TYPE)
+  .has('ogit/_out-type', OGIT_NODE_TYPE)
+  .inV()
+  .toString();
+
+client.gremlin(START_NODE_ID, query);
```

Comment:
Here we can add it as an example and get a ling to gremlin 3 documentation and hiro gremlin documentation. 
Btw there is `raw` step to add something our javascript implementation doesn't have. And it's actually possible pass to `client.gremlin` plain strings our gremlin is a just syntax helper